style.css

```diff
@@ -34,10 +38,10 @@
   header {
     margin-bottom: 95px;
```

Comment:
Hello @bernyhas ,
You do pretty well but unfortunately we request you to improve your css styles.

Happy coding